### PR TITLE
Add some craftable (makeshift) weapons

### PR DESCRIPTION
**Pull request by ErisMeleeEnjoyer**
## About The Pull Request
makeshift halberd
makeshift staff

makeshift staff is just a staff, has extended reach and screenshake, that's about it
makeshift halberd is a shitty version of holy halberd.

all of  these are craftable.
for more info check the code, i'm lazy after testing all this and space weather.
## Why It's Good For The Game
MANY people asked me to add these. So.
More craftable weapons is always good, luv me variety and vagabond guerilla warfare

## Changelog
:cl: Kegdo
add: makeshift halberd
add: makeshift staff
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
изображение

**Comment by Firefox13** [comment]
that doesn't looks like a sword

**Comment by ErisMeleeEnjoyer** [comment]
it's a chainsaw that a vagabond mutilated to turn into a weapon, it's not supposed to even *remotely* look like a sword

File: code/datums/craft/recipes/weapon.dm [hunk 1]
```diff
@@ -186,6 +186,42 @@
...
+	result = /obj/item/tool/hammer/staff
+	steps = list(
+		list(/obj/item/stack/rods, 1),
+		list(QUALITY_ADHESIVE, 10, "time" = 5),
+		list(/obj/item/stack/rods, 1),
+		list(QUALITY_ADHESIVE, 10, "time" = 5),
+		list(/obj/item/stack/rods, 1)
+	)
+
+/datum/craft_recipe/weapon/staff
```
Comment (Pink-Chink) [inline code/datums/craft/recipes/weapon.dm:200]:
```suggestion
/datum/craft_recipe/weapon/halberd
```

**Review by SirRichardFrancis — CHANGES_REQUESTED** [review]
Is that a chainsaw turned 90 degrees to the right?
It does look like bugged out chainsaw, certainly not a chainsaw sword.
100% chainsaw, 0% sword. If I could cockblock this addition - I would have.
And that's just a sprite, entire thing doesn't make any sense at all.

The rest is pretty cool.

**Comment by SirRichardFrancis** [comment]
Also "Vagabond did it" kinda meh explanation, they're not literal skavens, and if they were, this is still way too low effort.

**Comment by tristain01** [comment]
Gonna agree with Francis, that is a coder sprite moment.

**Comment by ValoTheValo** [comment]
this is just a chainsaw sprite flipped sideways

**Comment by Pink-Chink** [comment]
Staff and halberd are good. Chainsword needs a better sprite.

**Comment by ErisMeleeEnjoyer** [comment]
I'll sprite a proper chainsword next week then. Will remove it out of this PR cuz I don't have enough time to do it today

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**